### PR TITLE
4.6 update

### PR DESCRIPTION
**Pull request by SchSeba**
This PR updates the sriov-network-device-plugin base in the intel repo.

New features:
* support for accelerator devices
* switch to go mod
* add pciAddresses selector (VM case)

Misc:
* doc update
* unit test update

Note: ResourceConfig struct was updated to support both accelerator/network devices. Which also moves `isRdma` flag into Network device selectors (requires change in SR-IOV Network Operator).

### CODE REVIEW

File: Dockerfile.rhel7 [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.10 AS builder
+FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.13 AS builder
```
Comment (zshi-redhat) [inline Dockerfile.rhel7:1]: Is this change upstreamed?
what is the failure if not updating to go-1.13? like -modcache flag is not supported?

Comment (SchSeba) [reply Dockerfile.rhel7:1]: yes the issue is with -modcache in the clean

Comment (SchSeba) [reply Dockerfile.rhel7:1]: I didn't know we have this file on the u/s I will take a look and open the PR there

**Comment by zshi-redhat** [comment]
/lgtm
/approve
/hold
Feel free to unhold once SR-IOV Operator PR is ready.

**Pull request merged**